src/h5web/providers/Provider.tsx

```diff
@@ -23,6 +23,12 @@ function Provider(props: Props): JSX.Element {
         domain: api.domain,
         metadata,
         getValue: api.getValue.bind(api),
+        getValues: (ids) =>
+          Promise.all(
+            ids.map((id) =>
+              id ? api.getValue(id) : Promise.resolve(undefined)
```

Comment:
To keep the number of elements: https://github.com/silx-kit/h5web/pull/256/files#r509287208